src/probes/crash_probe.c

```diff
@@ -443,12 +444,18 @@ static bool is_banned_path(char *fullpath)
         }
 
         // Anything outside of /usr/, or in /usr/local/, we consider third-party
-        if (g_regex_match_simple("^[!/]usr[!/]", fullpath, 0, 0) == FALSE) {
-                return true;
-        } else if (g_regex_match_simple("^[!/]usr[!/]local[!/]", fullpath,
-                                        0, 0) == TRUE) {
-                return true;
-        }
+
+	if (strncmp(fullpath, "/usr/", 5) != 0)
```

Comment:
I agree with @fenrus75, the strstr/strncmp approach is simpler, and there is no need for the extra indirection.